spotbugs/etc/messages.xml

```diff
@@ -1321,6 +1321,14 @@ the subclass.</p>
 <![CDATA[
 <p>This detector looks for potential problems in implementing the Struts framework.
 </p>
+]]>
+    </Details>
+  </Detector>
+  <Detector class="edu.umd.cs.findbugs.detect.PublicAttributes">
+    <Details>
+<![CDATA[
+<p>This detector looks for public attributes that are also written by the metgods of the class.
```

Comment:
Misspelled "methods".

Reply:
This typo was fixed, so this conversation can be resolved.